test/python/test_pack.py

```diff
@@ -149,65 +150,81 @@ def rpm_archive_with_custom_units(module_tmpdir, light_project):
 def test_tgz_pack(tgz_archive, tmpdir):
     project = tgz_archive.project
 
+    extract_dir = os.path.join(tmpdir, 'extract')
+    os.makedirs(extract_dir)
+
     with tarfile.open(name=tgz_archive.filepath) as tgz_arch:
-        # usr/share/tarantool is added to coorectly run assert_filemodes
-        distribution_dir = os.path.join(tmpdir, 'usr/share/tarantool', project.name)
+        # usr/share/tarantool is added to correctly run assert_filemodes
+        distribution_dir = os.path.join(extract_dir, 'usr/share/tarantool', project.name)
         os.makedirs(distribution_dir, exist_ok=True)
 
-        tgz_arch.extractall(path=os.path.join(tmpdir, 'usr/share/tarantool'))
+        tgz_arch.extractall(path=os.path.join(extract_dir, 'usr/share/tarantool'))
         assert_distribution_dir_contents(
             dir_contents=recursive_listdir(distribution_dir),
             project=project
         )
 
         validate_version_file(project, distribution_dir)
-        assert_filemodes(project, tmpdir)
+        assert_filemodes(project, extract_dir)
 
 
 def test_rpm_pack(rpm_archive, tmpdir):
     project = rpm_archive.project
 
+    extract_dir = os.path.join(tmpdir, 'extract')
+    os.makedirs(extract_dir)
+
     ps = subprocess.Popen(
         ['rpm2cpio', rpm_archive.filepath], stdout=subprocess.PIPE)
-    subprocess.check_output(['cpio', '-idmv'], stdin=ps.stdout, cwd=tmpdir)
+    subprocess.check_output(['cpio', '-idmv'], stdin=ps.stdout, cwd=extract_dir)
     ps.wait()
     assert ps.returncode == 0, "Error during extracting files from rpm archive"
 
     if not tarantool_enterprise_is_used():
         assert_tarantool_dependency_rpm(rpm_archive.filepath)
 
-    check_package_files(project, tmpdir)
+    check_package_files(project, extract_dir)
     assert_files_mode_and_owner_rpm(project, rpm_archive.filepath)
 
+    # check rpm signature
+    cmd = [
+        'rpm', '--checksig', '-v', rpm_archive.filepath
+    ]
+    process = subprocess.run(cmd)
+    assert process.returncode == 0, "RPM signature isn't correct"
+
 
 def test_deb_pack(deb_archive, tmpdir):
     project = deb_archive.project
 
+    extract_dir = os.path.join(tmpdir, 'extract')
```

Comment:
Describe please

Reply:
You mean describing to you or adding a comment?)

Anyway, to correctly check package files archive, it should be extracted to the empty directory. Since I use `tmpdir` for all files used in test (project files etc.), I need to create subdirectory.

Reply:
Second one. Ok. However, it's not clear at first

Reply:
I added a comment, but I'm not sure that it's clear enough.